app/routes/company/components/CompanyDetail.tsx

```diff
@@ -1,19 +1,24 @@
 import moment from 'moment-timezone';
-import { Component } from 'react';
+import { useState } from 'react';
 import { Helmet } from 'react-helmet-async';
-import { Link } from 'react-router-dom';
-import { Content } from 'app/components/Content';
+import Button from 'app/components/Button';
+import {
+  Content,
+  ContentMain,
+  ContentSection,
+  ContentSidebar,
+} from 'app/components/Content';
+import DisplayContent from 'app/components/DisplayContent';
 import Icon from 'app/components/Icon';
-import { Image } from 'app/components/Image';
-import InfoBubble from 'app/components/InfoBubble';
+import { Flex } from 'app/components/Layout';
 import LoadingIndicator from 'app/components/LoadingIndicator';
 import NavigationTab from 'app/components/NavigationTab';
-import { jobType, Year } from 'app/routes/joblistings/components/Items';
-import EventItem from 'app/routes/overview/components/EventItem';
-import { renderMeta } from 'app/routes/overview/components/utils';
+import { JoblistingItem } from 'app/routes/joblistings/components/JoblistingList';
+import { ListEvents } from 'app/routes/users/components/UserProfile';
```

Comment:
Hmm, I'd prefer if you don't mix components from other routes. 

This component is also just a `map` that returns `EventItem` components from `app/components/EventItem`, so I'd use that directly. Or move the EventList  into app/components and import that here and in the userprofile.